devicetypes/gs/gatorsystem-homewatcher.src/gatorsystem-homewatcher.groovy

```diff
@@ -0,0 +1,191 @@
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+
+
+metadata {
+        definition (name: "GatorSystem HomeWatcher", namespace: "GS", author: "GS_coder000") {
+
+	capability "Motion Sensor"
+	capability "Battery"
+	capability "Contact Sensor"
+    capability "Presence Sensor"
+      
+    command "stopMotion"  
+    command "resetOccupancy"
+      
+        fingerprint profileId: "0104", deviceId: "0402", inClusters: "0000, 0500", outClusters: "0502", manufacturer: "GatorSystem", model: "GSHW01"
```

Comment:
1. deviceJoinName is missing in this fingerprint
Please add deviceJoinName: "GatorSystem Multipurpose Sensor" to the fingerprint
2. Please remove profileId, deviceId, inClusters, outClusters from the fingerprint
3. Please add "Raw Description" above this finger print in a comment
(It can be found at graph.api.smartthings.com ->  My Devices -> Raw Description) so it would look like:
// Raw Description .... 
fingerprint ....

Reply:
Changed.